### ISSUE TITLE
[BUG][UI5-List]Focus indicator is missing for the icon in ui5-list

### ISSUE THREAD

**Issue by sfsf-xweb-sh** [SAP/ui5-webcomponents#2961]
**Dear colleagues,** 
I am from SuccessFactors Web Experience Shanghai team.
 My name is Hubery Shen and this account is our team account.  

Bug Description:
Focus indicator is missing for the icon in ui5-list
![Screen Shot 2021-03-18 at 1 18 21 PM](https://user-images.githubusercontent.com/78520316/111577461-46d07e80-87ed-11eb-9844-2630172c1600.png)

**Comment by ilhan007** [comment]
Hello, the icon is not supposed to get focused by design. It's meant for indication. What is the requirement, you need  a focus to execute different action and react on click/press or for accessibility reasons?

**Comment by sfsf-xweb-sh** [comment]
Hi @ilhan007 

QAS team told us this icon should be focusable

**Comment by ilhan007** [comment]
Hello @sfsf-xweb-sh

could you please provide the UX or A11y specification of the ListItem that states that, because I could not it.


Also, you can test this exmaple of the sap.m.List with sap.m.StandardListitem, 
https://openui5nightly.hana.ondemand.com/entity/sap.m.StandardListItem/sample/sap.m.sample.StandardListItemNavigated

Set the icon property of an arbitrary list item and you can see that the icon has only decorative purpose, can't be focused.

**Comment by ilhan007** [comment]
Closing due to inactivity.
